src/Command/Command.php

```diff
@@ -0,0 +1,214 @@
+        }
+
+        try {
+            $configFile = Yaml::parse(file_get_contents($this->getPath().'/'.self::CONFIG_FILE));
+            $config = array_replace_recursive($configFile, $config);
+            $this->builder = (new Builder($config, $this->messageCallback($output)))
+                ->setSourceDir($this->getPath())
+                ->setDestinationDir($this->getPath());
+        } catch (ParseException $e) {
+            throw new \Exception(sprintf('Config file parse error: %s', $e->getMessage()));
+        } catch (\Exception $e) {
+            throw new \Exception(sprintf($e->getMessage()));
+        }
+
+        return $this->builder;
+    }
+
+    /**
+     * Create the Progress bar.
+     *
+     * @param OutputInterface $output
+     * @param int             $max
+     */
+    protected function createProgressBar(OutputInterface $output, $max)
+    {
+        if ($this->progressBar === null || $max != $this->progressBarMax) {
+            $this->progressBarMax = $max;
+            $this->progressBar = new ProgressBar($output, $max);
+            $this->progressBar->setOverwrite(true);
+            $this->progressBar->setFormat(' %percent:3s%% [%bar%] %current%/%max%');
+            $this->progressBar->setBarCharacter('#');
+            $this->progressBar->setEmptyBarCharacter(' ');
+            $this->progressBar->setProgressCharacter('#');
+            $this->progressBar->setRedrawFrequency(1);
+            $this->progressBar->start();
+        }
+    }
+
+    /**
+     * Return Progress Bar.
+     *
+     * @return ProgressBar
+     */
+    protected function getProgressBar()
+    {
+        return $this->progressBar;
+    }
+
+    /**
+     * Print the Progress Bar.
+     *
+     * @param OutputInterface $output
+     * @param int             $itemsCount
+     * @param int             $itemsMax
+     */
+    protected function printProgressBar(OutputInterface $output, $itemsCount, $itemsMax)
+    {
+        $this->createProgressBar($output, $itemsMax);
+        $this->getProgressBar()->clear();
+        $this->getProgressBar()->setProgress($itemsCount);
+        $this->getProgressBar()->display();
+        if ($itemsCount == $itemsMax) {
+            $this->getProgressBar()->finish();
+            $output->writeln('');
+        }
+    }
+
+    /**
+     * Custom message callback function.
+     *
+     * @param OutputInterface $output
+     */
+    public function messageCallback(OutputInterface $output)
+    {
+        return function ($code, $message = '', $itemsCount = 0, $itemsMax = 0) use ($output) {
+            if (strpos($code, '_PROGRESS') !== false) {
+                if ($output->isVerbose()) {
+                    if ($itemsCount > 0) {
+                        $output->writeln(sprintf(' (%u/%u) %s', $itemsCount, $itemsMax, $message));
+
+                        return;
+                    }
+                } else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method messageCallback uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/manual/Narno/Cecil/pullRequest?prid=4722362)